src/Parser/Core/Modules/Items/Healthstone.js

```diff
@@ -0,0 +1,55 @@
+import SPELLS from 'common/SPELLS';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Abilities from 'Parser/Core/Modules/Abilities';
+import SpellUsable from 'Parser/Core/Modules/SpellUsable';
+
+const HEALTHSTONE_SPELLS = [
+  SPELLS.HEALTHSTONE,
+  SPELLS.ANCIENT_HEALING_POTION,
+  SPELLS.ASTRAL_HEALING_POTION,
+];
+
+const COOLDOWN_MS = 3600000; // one hour
+const OUT_OF_COMBAT_COOLDOWN_MS = 60000; // one minute
```

Comment:
In that case please leave a comment explaining how this works ingame (or preferably add a test for this requirement).

Also if it’s triggered by dying, how about  naming it so that it refers to that instead of leaving combat?